configuration/configuration.py

```diff
@@ -6,11 +6,11 @@
 # Based on https://github.com/netbox-community/netbox/blob/develop/netbox/netbox/configuration.example.py
 
 # Read secret from file
-def read_secret(secret_name):
+def read_secret(secret_name, default=''):
```

Comment:
```suggestion
def read_secret(secret_name, default=None):
```

Is this possible? I think it's no good to have the default value of `default` be an empty string.

Reply:
I now see that you've only copied what was there. Still, if you don't mind change it to `None`, that'd be great :)